package.json

```diff
@@ -44,10 +45,7 @@
     "test:integration:sequential": "mocha test/integration/**/staging/sequential/*.test.ts --timeout 10000000",
     "watch:build": "tsc -p tsconfig.json -w",
     "watch:test": "mocha --watch test/**/*.test.ts",
-    "local-meta": "curl -H 'Content-Type: application/json' -d '{\"id\":\"1\", \"jsonrpc\":\"2.0\", \"method\": \"state_getMetadata\", \"params\":[]}' http://localhost:9933 > src/json/parachain.json",
-    "docker-parachain-start": "ts-node scripts/setup-parachain-docker.ts",
-    "docker-parachain-stop": "docker-compose -f local-setup/docker-compose.yml down -v",
-    "set": "npx ts-node src/utils/set-state/set-state.ts"
+    "local-meta": "curl -H 'Content-Type: application/json' -d '{\"id\":\"1\", \"jsonrpc\":\"2.0\", \"method\": \"state_getMetadata\", \"params\":[]}' http://localhost:9933 > src/json/parachain.json"
```

Comment:
Suggestion (not blocking): reference using that in the README instead instead of the `curl` command? (Or maybe alongside that)